tfjs/src/index_test.ts

```diff
@@ -52,12 +52,10 @@ describe('Union package', () => {
   });
 
   it('has cpu backend', () => {
-    const backend = tf.findBackend('cpu');
-    expect(backend).not.toBeNull();
+    expect(tf.MathBackendCPU).not.toBeNull();
   });
 
   it('has webgl backend', () => {
-    const backend = tf.findBackend('webgl');
-    expect(backend).not.toBeNull();
+    expect(tf.MathBackendWebGL).not.toBeNull();
```

Comment:
We should still run the original check, I think. It's checking if the cpu and webgl backends have been registered, whereas the new check is checking if their constructors are exported.

Reply:
Make senses. Thank you!